keep-ui/entities/alerts/model/useAvailableAlertFields.ts

```
@@ -0,0 +1,48 @@
+  };
+  const { data: alertsFound = [], isLoading } = useSearchAlerts({
+    query: defaultQuery,
+    timeframe,
+  });
+
+  const fields = useMemo(() => {
+    const getNestedKeys = (obj: any, prefix = ""): string[] => {
+      return Object.entries(obj).reduce<string[]>((acc, [key, value]) => {
+        const newKey = prefix ? `${prefix}.${key}` : key;
+        if (value && typeof value === "object" && !Array.isArray(value)) {
+          return [...acc, ...getNestedKeys(value, newKey)];
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)